lib/jxl/enc_frame.cc

```diff
@@ -253,15 +253,16 @@ uint64_t FrameFlagsFromParams(const CompressParams& cparams) {
   return flags;
 }
 
-Status LoopFilterFromParams(const CompressParams& cparams,
+Status LoopFilterFromParams(const CompressParams& cparams, bool streaming_mode,
                             FrameHeader* JXL_RESTRICT frame_header) {
   LoopFilter* loop_filter = &frame_header->loop_filter;
 
   // Gaborish defaults to enabled in Hare or slower.
-  loop_filter->gab = ApplyOverride(
-      cparams.gaborish, cparams.speed_tier <= SpeedTier::kHare &&
-                            frame_header->encoding == FrameEncoding::kVarDCT &&
-                            cparams.decoding_speed_tier < 4);
+  loop_filter->gab =
```

Comment:
TODO to remove streaming_mode from here?

Reply:
Done.